src/app/app.routing.ts

```diff
@@ -1,4 +1,5 @@
 import { Routes } from '@angular/router';
+import { AuthGuard } from './services/authguard.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/SimpleSurveyProject/SimpleSurvey-Frontend/pullRequest?prid=7430210)
```suggestion
import { AuthGuard } from "./services/authguard.service";
```